### ISSUE TITLE
Dudas text-search

### ISSUE THREAD

**Issue by juansjimenez** [IIC2413/Syllabus-2020-1#346]
Hola!
Hay algo que no me queda muy claro respecto a como poder agregar la búsqueda incluyendo las frases deseadas. Cuando se trata de las requeridas, se tiene que tener toda la frase completa en los mensajes que se retornen, sin embargo, para las deseadas, supongo que basta con incluir cada palabra de cada frase con sentencias de or dentro del search? Lo pregunto ya que no conozco una forma de hacer or de frases enteras. Por ende, debería bastar, por ejemplo si tengo dos frases deseables: **Hola como estas, bien y tu**, con incluir en el search simplemente "hola como estas bien y tu"???

Gracias!

**Comment by nivek0o0** [comment]
Sí, basta con que cada palabra de cada frase deseada esté presente en el `text-search`
